channeldb/channel.go

```diff
@@ -293,6 +295,12 @@ func (c *OpenChannel) UpdateCommitment(newCommitment *wire.MsgTx,
 		if err := putChanCapacity(chanBucket, c); err != nil {
 			return err
 		}
+		if err := putChanSatoshisSent(chanBucket, c); err != nil {
```

Comment:
Ah, I see. Yeah I'm down with renaming the function in order to add more clarity w.r.t exactly what it's doing.

Also I agree all the serialization code in this file could use a good bit of refactoring to clean it up and reduce all the lingering duplication strewn around. I have a few TODO's strewn around the package which describe steps I've thought of that could be taken to improve the quality of the code in the package, but I haven't yet got around to implementing them all.